examples/with-doge/src/createDogeTx.ts

```diff
@@ -0,0 +1,254 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+import axios from "axios";
+import * as bitcoin from "bitcoinjs-lib";
+import { Turnkey as TurnkeyServerSDK } from "@turnkey/sdk-server";
+
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+// Dogecoin mainnet P2PKH (starts with 'D')
+const SIGN_WITH = process.env.SIGNER_ADDRESS;
+const DEST_ADDRESS = process.env.DEST_ADDRESS;
+
+// 33-byte compressed public key hex for SIGNER_ADDRESS
+const COMPRESSED_PUBKEY_HEX = process.env.SIGNER_ADDRESS_PUBKEY;
+
+// Send 1 Doge with a 0.004 tx fee
+const SEND_AMOUNT_SATS = 1e8;
+const FEE_SATS = 400_000;
+
+// Optional: manually preseed UTXOs; leave [] to auto-fetch
+type UTXO = { txid: string; vout: number; valueSats: number };
+const PRESEEDED_UTXOS: UTXO[] = [
+  // { txid: '...', vout: 0, valueSats: 123_456_789 },
+];
+
+// Dogecoin mainnet (legacy P2PKH only)
+const doge: bitcoin.networks.Network = {
+  messagePrefix: "\x19Dogecoin Signed Message:\n",
+  bech32: undefined as any,
+  bip32: { public: 0x02facafd, private: 0x02fac398 },
+  pubKeyHash: 0x1e,
+  scriptHash: 0x16,
+  wif: 0x9e,
```

Comment:
every one of these lines deserves a comment IMO, because it's not obvious what these values are. We should reference the official documentation wherever possible.